collectors/test-results/jira-xray/src/main/java/com/capitalone/dashboard/core/client/testexecution/TestExecutionClientImpl.java

```diff
@@ -26,6 +26,7 @@
 import com.capitalone.dashboard.util.FeatureCollectorConstants;
 import com.google.common.collect.Lists;
 import org.slf4j.LoggerFactory;
+import org.springframework.util.CollectionUtils;
```

Comment:
Any specific reason to use Spring CollectionUtils in place of Apache Commons?

Reply:
No specific reason.